oras/provider.py

```diff
@@ -219,20 +219,23 @@ def upload_blob(
         container: container_type,
         layer: dict,
         do_chunked: bool = False,
+        chunk_size=None,
```

Comment:
This should default to the default chunk size instead of None. If it's a shared value, put in defaults.py and reference from both / several function inputs.